_includes/master/charts/calico/values.yaml

```diff
@@ -0,0 +1,32 @@
+calicoctl:
+  image: quay.io/calico/calicoctl
+  tag: master
+node:
+  image: quay.io/calico/node
```

Comment:
It would be really good if we could take this opportunity to ditch the "quay" on these guys. We've got several attempts in progress to do this which likely conflict with this PR.

We obvs can do as a follow-up.

Reply:
I think I've seen one of those attempts that pretty simply moves the `quay.io/` to another variable and then substitutes that in here.

If we're minded now to accept one of those suggestions, I guess it would be friendlier to do that first (and then we rebase) than to merge this first and ask them to rebase.

Reply:
Yeah, I agree. 

I'd like to merge https://github.com/projectcalico/calico/pull/2364, but it needs rebase first.